src/api/parser/src/utils/__mocks__/indexer.js

```diff
@@ -0,0 +1,64 @@
+// Mock storage for our es data
```

Comment:
@rclee91 and @JiaHua-Zou are the elastic-mock-wizards, let's get their review on this part.

Reply:
I've been trying to look at it. But since there's no tests I can't really tell if it will work or not. 
As for mocking with ElasticSearch, it doesn't look like we need a __mocks__ folder at all, everything is done with `mock.add`.